functions/menu.bash

```diff
@@ -136,13 +136,15 @@ show_main_menu() {
     "   | Disable NTP"            "Disable time synchronization via systemd-timesyncd to NTP servers" \
     "34 | Change passwords"       "Change passwords for Samba, openHAB Console or the system user" \
     "35 | Serial port"            "Prepare serial ports for peripherals like Razberry, SCC, Pine64 ZWave, ..." \
-    "36 | WiFi setup"             "Configure wireless network connection" \
+    "36 | Disable framebuffer"    "Disable framebuffer on RPi to minimize memory usage" \
+    "   | Enable framebuffer"     "Enable framebuffer (standard setting)" \
+    "37 | WiFi setup"             "Configure wireless network connection" \
     "   | Disable WiFi"           "Disable wireless network connection" \
-    "37 | Move root to USB"       "Move the system root from the SD card to a USB device (SSD or stick)" \
     "38 | Use zram"               "Use compressed RAM/disk sync for active directories to avoid SD card corruption" \
     "   | Uninstall zram"         "Don't use compressed memory (back to standard Raspberry Pi OS filesystem layout)" \
-    "39 | Setup Exim Mail Relay"  "Install Exim4 to relay mails via public email provider" \
-    "3A | Setup Tailscale VPN"    "Establish or join a WireGuard based VPN using the Tailscale service" \
+    "39 | Move root to USB"       "Move the system root from the SD card to a USB device (SSD or stick)" \
```

Comment:
Not really related, but do we really still want to support this?

Reply:
I just renumbered options but wanted to keep zram 38 so I don't have to change docs that may reference this.